components/GymPlan.js

```diff
@@ -0,0 +1,156 @@
+import styled from "styled-components";
+
+export default function GymPlan() {
+  return (
+    <>
+      <StyledHeadline> Gym Plan </StyledHeadline>
+      <StyledSection>
+        <StyledHeadlineSection> Warm-Up </StyledHeadlineSection>
+        <form>
```

Comment:
As I see it you writing 5x the same form. 
Why don't make it a component and use props. You can reduce your code a lot and you save time when changing something. DRY ;)